### ISSUE TITLE
mod : kill bots command

### ISSUE THREAD

**Issue by Yeldur** [namidaka/crpg#333]
**Is your feature request related to a problem? Please describe.**
Bots with bad navmeshing on maps frequently end up running into areas where they can't do anything and sit there for entire rounds

**Describe the solution you'd like**
A kill bots command

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Muparadzi** [comment]
@Yeldur Is a kill all agents command more appropriate for this? I have concerns about the implications of allowing us to kill all bots on modes like DTV.

**Comment by Yeldur** [comment]
> @Yeldur Is a kill all agents command more appropriate for this? I have concerns about the implications of allowing us to kill all bots on modes like DTV.

Yes and no. If I'm thinking correctly 'All Agents' would refer to players as well, so if for example, if somebody was to try using this on DTV to clear bots, this would result in the map ending. 

If the potential problem point is DTV, is there a way to blacklist DTV as a server from being able to execute this command?

**Comment by Muparadzi** [comment]
Killing all bots kills the viscount which would end dtv anyway. But a kill all command is more versatile than one that only kills bots.

**Comment by Yeldur** [comment]
Fair point. I'd go with that then, it's probably going to be a rare-use command anyways as if we **did** need to kill a player we do have a !kill command already that works off UID or USN
